CHANGELOG.md

```diff
@@ -9,17 +9,21 @@
 -->
 
 ## master
+* Refactor shared code from Violation and Markdown into a base class [@telyn][@telyn] [#1186](https://github.com/danger/danger/pull/1184)
+* Add <=> methods to Violation and Markdown [@telyn][@telyn] [#1186](https://github.com/danger/danger/pull/1184)
+* Add sketch of MessageGroup and a totally unimplemented MessageAggregator, for the coming PR-host-agnostic
+  inline-comment grouping support [@telyn][@telyn] [#1184](https://github.com/danger/danger/pull/1184)
 
 ## 6.2.2
 
-* Fix incorrect method call in Bitbucket Cloud arising from my own changes in 6.2.1 [@telyn](https://github.com/telyn) [#1184](https://github.com/danger/danger/pull/1184)
+* Fix incorrect method call in Bitbucket Cloud arising from my own changes in 6.2.1 [@telyn][@telyn] [#1184](https://github.com/danger/danger/pull/1184)
 
 ## 6.2.1
 
 * Fix Bitbucket Pipelines documentation
 * Update Ruby-Git to 1.6.0 to fix output encoding
-* Add inline-comments support to Bitbucket Cloud [@telyn](https://github.com/telyn) [#1182](https://github.com/danger/danger/pull/1182)
-* Make emojis display better on Bitbucket Cloud [@telyn](https://github.com/telyn) [#1182](https://github.com/danger/danger/pull/1182)
+* Add inline-comments support to Bitbucket Cloud [@telyn][@telyn] [#1182](https://github.com/danger/danger/pull/1182)
+* Make emojis display better on Bitbucket Cloud [@telyn][@telyn] [#1182](https://github.com/danger/danger/pull/1182)
```

Comment:
If you leave this blank, then markdown will assume it is the same as the text on the left -`[@telyn][]` === `[@telyn][@telyn]`

Reply:
oh, neat, didn't know that 😁